app/components/PlaylistCover.tsx

```diff
@@ -0,0 +1,28 @@
+import { Song } from '@app/models/music'
+import CoverArt from '@app/components/CoverArt'
+import React from 'react'
+import { StyleSheet, ImageResizeMode, FlatList } from 'react-native'
+
+export function genPlaylistCover2x2Component(_songs: Song[]) {
```

Comment:
Since this returns a component it should follow the same pattern/rules as other react function components, specifically in naming and the way it's being memoized/exported to match other components in the project.  See [Star.tsx](https://github.com/austinried/subtracks/blob/master/app/components/Star.tsx) for an example.  For a name I would suggest `PlaylistCoverArt2x2`.

Overall though I think this should be moved into `CoverArt`, which already specializes itself based on if it's being passed an artist or "cover" type props, which could be specialized further to take playlist type props and display the grid if the data is available.